com.unity.perception/Runtime/GroundTruth/Labelers/CameraLabeler.cs

```diff
@@ -80,14 +64,25 @@ public abstract class CameraLabeler
         /// Called just before the camera renders each frame the the labeler is enabled and <see cref="SensorHandle.ShouldCaptureThisFrame"/> is true.
         /// </summary>
         protected virtual void OnBeginRendering() {}
+        /// <summary>
+        /// Labeling pass to display labeler's visualization components, if applicable. Important note, all labeler's visualizations need
+        /// to use Unity's Immediate Mode GUI (IMGUI) <see cref="https://docs.unity3d.com/Manual/GUIScriptingGuide.html"/> system.
+        /// This called is triggered from <see cref="perceptionCamera.OnGUI"/> call.
+        /// </summary>
+        protected virtual void OnVisualize() {}
+        /// <summary>
+        /// In this pass, a labeler can add custom GUI controls to the scene. Important note, all labeler's additional
```

Comment:
It would be helpful to explain the timing between OnVisualize and OnVisualizeAdditionalUI a bit more. Like, "OnVisualizeAdditionalUI is called after ..."

Reply:
Updated documentation.